### DISCUSSION TITLE
Bounding box for group doesn't update when grouped objects stroked width is changed are changed

### DISCUSSION THREAD

**Discussion by sohailahmad07** [fabricjs/fabric.js#9973] [Q&A] [answered]
Hi i have a group with a shape and a text box and when i try to change the stroke with the bounding box, it does not get updated

here a video
![Screen Capture_select-area_20240704222936](https://github.com/fabricjs/fabric.js/assets/58299055/d7808165-1e9b-424a-837d-cab3c0399815)

and here and a codepen you guys can test it i am using fabric 6.0

in pen i am using 5.3 because 6.0 was not available 

https://codepen.io/Sohail-Ahmad-the-solid/pen/abgoWLo

**Comment by sohailahmad07** [comment, marked as answer]
For anyone having this issue in the new 6.0 you can call ` group.triggerLayout();` and i will fix the issue
after update the stroke and i will fix the issue here updated code pen


```
// update group object stroke like so 

 group.item(0).set({
        strokeWidth: Number(e.target.value)
      })
    
group.triggerLayout();

canvas.requestRenderAll()    
 
```

**Reply by yashChapaniOB** [reply]
Superb, Thank you so much it saves me lots of manual calculation
